static/app/views/nav/orgDropdown.tsx

```diff
@@ -187,16 +187,9 @@ export function OrgDropdown({className}: {className?: string}) {
   );
 }
 
-const OrgDropdownTrigger = styled(Button)<{isMobile: boolean}>`
-  height: 44px;
-  width: 44px;
-
-  ${p =>
-    p.isMobile &&
-    css`
-      width: 32px;
-      height: 32px;
-    `}
+const OrgDropdownTrigger = styled(Button)<{width: number}>`
+  height: ${p => p.width}px;
+  width: ${p => p.width}px;
```

Comment:
there’s a `toPixels` util for this:

https://github.com/getsentry/sentry/blob/54e40da08a42fce9d68109a6fa86eed85fac4707/static/app/utils/number/toPixels.tsx#L7-L9

Reply:
> that passes a string value that isn't px

generally agree, but this should be caught by TypeScript, as `CssSize` has good types:

```
export type CssSize = number | `${number}px` | `1em` | '100%';
```

so you shouldn’t be allowed to pass arbitrary strings in.

but I also agree that it isn’t needed here as you know you have a `number` and this function is for cases where you don’t know if you have a number value or a value with `px` at the end already